src/sider/senoppfolging/SenOppfolging.tsx

```diff
@@ -27,6 +29,8 @@ const texts = {
 export default function SenOppfolging(): ReactElement {
   const { data: svar } = useSenOppfolgingSvarQuery();
   const svardato = svar && tilLesbarDatoMedArUtenManedNavn(svar.createdAt);
+  const { data: kandidater } = useSenOppfolgingKandidatQuery();
+  const kandidat = kandidater[0];
```

Comment:
Hvis dataen ikke er lastet inn enda/finnes, så vil vel denne gi en feil?

Reply:
Tror query-hooken defaulter til tom liste hvis `data` er `undefined` så skal gå bra :+1:

Reply:
Aha, ja for indeks-aksessering på en tom liste gir bare `undefined` ja 🤔 Tenkte det kræsja.